### PR TITLE
Konvoy docs latest

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket
n/a

## Description of changes being made
Updating docs for backporting releases. 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.8/install/install-airgapped/index.md [hunk 1]
```diff
@@ -293,6 +293,18 @@ For example on `Ubuntu 18.04`, you must replace the following Deb packages and t
     - libnspr4_2%3a4.18-1ubuntu1_amd64.deb
     - libnss3_2%3a3.35-2ubuntu2.12_amd64.deb
 
+### Enterprise Linux 8 Airgapped Packages
```
Comment (cmastr) [inline pages/dkp/konvoy/1.8/install/install-airgapped/index.md:296]: @jr0d has this been released yet?

**Review by cmastr — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/1.7/prerequisites/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites
```
Comment (cmastr) [inline pages/dkp/konvoy/1.7/prerequisites/index.md:4]: confusing as to why we have this now?

Comment (dkoshkin) [reply pages/dkp/konvoy/1.7/prerequisites/index.md:4]: This was merged a few months ago https://github.com/mesosphere/konvoy/pull/2951

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ spec:
         - name: kommander
           enabled: true
   ...
-  version: v1.8.0
+  version: v1.7.0
```
Comment (msdolbey) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:93]:
```suggestion
  version: v1.8.0
```

File: pages/dkp/konvoy/1.7/prerequisites/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/konvoy/1.7/prerequisites/index.md:4]: ok thanks for the context.

**Review by cmastr — APPROVED** [review]

**Pull request merged**